dev/system-test/firestore.ts

```diff
@@ -2265,6 +2280,40 @@ describe('QuerySnapshot class', () => {
   });
 });
 
+describe('BulkWriter class', () => {
+  let firestore: Firestore;
+  let randomCol: CollectionReference;
+
+  beforeEach(() => {
+    firestore = new Firestore({});
+    randomCol = getTestRoot(firestore);
+  });
+
+  afterEach(() => verifyInstance(firestore));
+
+  it.skip('increments and decrements the pendingOperationsCount', async () => {
```

Comment:
What about something like "can terminate() when BulkWrite is closed"?

Reply:
done.